ui/de-webapp/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+multipart.max-file-size=10000000
+multipart.max-request-size=10000000
```

Comment:
If it's just for local dev but might stick around, please also add it to a `.gitignore` file somewhere or other!